src/loadgenerator/locustfile.py

```diff
@@ -78,5 +78,5 @@ def on_start(self):
 
 class WebsiteUser(HttpLocust):
     task_set = UserBehavior
-    min_wait = 1000
-    max_wait = 10000
+    min_wait = 1000 * 60
+    max_wait = 10000 * 60
```

Comment:
It seems like this will make very few requests since it would wait between 1min and 10min so on average 3 req/5min or so. I think we want to have the graph of traces show a little more data.

Maybe reduce to ~1 req/10 sec so maybe:
USERS = 3
min_wait = 1000 * 15
max_wait = 1000 * 45

WDYT?

Reply:
Then I updated the numbers as the followings:
USERS = 5
min_wait = 1000 * 10 (10 sec)
max_wait = 1000 * 30 (30 sec)

And instead of increasing the number of requests, I changed the test scenarios not to cause error reporting. The points are:
* Currencies to select (removed 'CAD' from the candidates)
* Numbers of items to add to cart (in the scenario, it adds items 2 times, and I set the maximum to 5 for each times)
* Email address for the shipping. (changed it to 'next@tokyo.jp')